.github/workflows/semantic-pr.yml

```diff
@@ -0,0 +1,21 @@
+name: Semantic PR
```

Comment:
nit: `Check semantic PR titles`

also -- will it look at description also? or just the title?

Reply:
Just the title. Updated